index.markdown

```diff
@@ -211,52 +211,52 @@ redirect_from: '/moveit/'
                     <h2 class="h1 text-center">Current and Future Versions</h2>
                 </div>
                 <div class='col-sm-12 time-line-wrappper'>
-                    <h3>Melodic LTS</h3>
+                    <h3>MoveIt 1.1 Noetic</h3>
                     <div class="time-line">
                         <div class="time-line--orange" id="time-line--orange-1"></div>
-                        <div class="time-line--blue"></div>
+                        <div class="time-line--blue" id="time-line--blue-1"></div>
                         <div class="time-line--red" id="time-line--red-1"></div>
                     </div>
                     <div class="time-line-content">
                         <div class="time-line--info">
-                            <p>23 May 2018</p>
+                            <p>March 2019</p>
                         </div>
                         <div class="time-line--info">
                             <span class="time-line-box time-line-box--blue"></span>
                             <p>
                                 Initial Development
-                                <small>23 May 2018</small>
+                                <small>1 March 2019</small>
                             </p>
                         </div>
                         <div class="time-line--info">
                             <span class="time-line-box time-line-box--orange"></span>
                             <p>
                                 API Freeze
-                                <small>March 2019</small>
+                                <small>1 February 2020</small>
                             </p>
                         </div>
                         <div class="time-line--info">
                             <span class="time-line-box time-line-box--red"></span>
                             <p>
-                                Official 1.0 Release
-                                <small>April 2019</small>
+                                Official Release
+                                <small>1 March 2020</small>
```

Comment:
I couldn't find Noetic release date but I assume releasing on March 1, 2020 works.

Reply:
yea its not firm, but we needed something for the progress bar graphic